packages/docs/getting-started/using-metamask-with-celo/README.md

```diff
@@ -1,32 +1,44 @@
 # Using MetaMask with Celo
 
-{% hint style="warning" %}
-MetaMask is primarily used for interacting with the Ethereum blockchain, and does not natively support Celo compatibility. As such, we recommend only sophisticated users and developers use it, at your own risk. Please check out options for Celo native wallets [here](https://docs.celo.org/getting-started/wallets).
-{% endhint %}
-
 {% hint style="danger" %}
-Do not send ETH to your Celo address. Do not send CELO assets to your Ethereum address.
+Do not send ETH to your Celo address. Do not send CELO assets to your Ethereum address. Always make sure that you are connected to the correct network.
 {% endhint %}
 
-[MetaMask](https://metamask.io/) is a crypto wallet that can be used in-browser and on mobile to interact with the Ethereum blockchain. Many dApps in the space integrate with MetaMask, and we're excited to bring its functionality to the Celo ecosystem.
+[MetaMask](https://metamask.io/) is a crypto wallet that can be used in a web browser and on mobile devices to interact with the Ethereum blockchain. Many dApps in the space integrate with MetaMask, and we're excited to bring its functionality to the Celo ecosystem.
+
+Since Celo network's [Donut Hardfork](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0027.md), activated on Mainnet on May 19th, 2021, the protocol now supports [Ethereum-compatible transactions](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0035.md). This means that users may use MetaMask to interact with the Celo blockchain and dApp developers can more easily port Ethereum dApps to the Celo blockchain.
+
+# **How to use MetaMask with Celo**
+
+### **For end users:**
 
-With the Celo network's [Donut Hardfork](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0027.md), which was activated on Mainnet on May 19th, 2021, the protocol now supports [Ethereum-compatible transactions](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0035.md). This means that users are now able to use MetaMask to interact with the Celo blockchain and dApp developers can more easily port over Ethereum dApps onto Celo. The following guide aims to detail step-by-step how to do that.
+* [Configure a MetaMask Desktop or Web Wallet to Work with Celo](manual-setup.md)
+* [Setup a Ledger to Work with MetaMask to Work With Celo](packages/docs/getting-started/using-metamask-with-celo/using-a-ledger-with-metamask.md)
```

Comment:
these relative links are broken. the path is relative to the current file, so `packages/docs/getting-started/using-metamask-with-celo/using-a-ledger-with-metamask.md` should be `using-a-ledger-with-metamask.md`. You can check to make sure links are working properly by previewing the file and clicking the links. Broken links will give you a 404.

Reply:
Sorry about that updated links